index.html

```diff
@@ -0,0 +1,601 @@
+             title="mypages"></i></a></li>
+    </ul>
+  </nav>
+
+  <!-- on mobile navbar is after Header -->
+
+  <nav class="mobile-nav"
+       aria-label="Main navigation for mobile devices">
+    <div class="mobilenav-left">
+      <div class="ham-menu">
+        <span></span>
+        <span></span>
+        <span></span>
+      </div>
+    </div>
+    <ul class="mobilenav-right">
+      <li><a href="#"><i class="fa-solid fa-magnifying-glass"></i></a></li>
+      <li><a href="#"><i class="fa-solid fa-bag-shopping"></i></a></li>
+      <li><a href="#"><i class="fa-solid fa-spaghetti-monster-flying"></i></a></li>
+    </ul>
+  </nav>
+
+  <!-- Header video  on mobile and hero image on landscape/ large screnens -->
+  <header class="small-header"
+          aria-label="Header with video  only on mobile">
+    <div class="hero-video">
+      <video controls
+             autoplay
+             loop
+             muted
+             playsinline
+             class="hero-video"
+             aria-label="Introduction scene from Dead Men Walk">
+        <source src="./images/dead-men-walk-intro.mp4"
+                type="video/mp4">
+        Your browser does not support the video tag.
+      </video>
+
+      <div class="hero-shadow"></div>
+    </div>
+    <div class="neon-text">
+      <h1 data-text="MIDNIGHT MATINÉE">MIDNIGHT <br> MATINÉE
+      </h1>
+    </div>
+  </header>
+
+  <header class="big-header"
+          aria-label="Header with image for large screens">
+    <img id="hero-image"
+         src="./images/theatre.webp"
+         alt="old and creepy Black & white theatre">
+    <div class="hero-shadow"></div>
+    <div class="neon-text">
+      <h1 data-text="MIDNIGHT MATINÉE">MIDNIGHT <br> MATINÉE
+      </h1>
+    </div>
+  </header>
+
+  <!-- hamburger list items -->
+
+  <div class="ham-offscreen"
+       aria-label="Hamburger menu items">
+    <ul>
+      <li>Home</li>
+      <li>Login</li>
+      <li>Make A Request</li>
+      <li>Newsletter</li>
+      <li>Forum</li>
+    </ul>
+  </div>
+
+  <!--Main Content -->
+
+
+  <main aria-label="Main content area">
+    <article>
+      <h2 id="MM-title"
+          style="letter-spacing: 3px;">
+        I BID YOU WELCOME... </h2>
+      <h5 Class="subheader-text"
+          style="letter-spacing: 1px;">
+        Where the shadows whisper and the legends live on! Step into our crypt of classic horror movie posters,
+        unearthed from the golden age of terror spanning the eerie realms of the 1930s to the 1960s. From the fanged
+        menace of Dracula to the towering dread of Godzilla, our collection is a shrine to the macabre. Whether you're
+        a devoted horror fan or seeking a spine-chilling gift, you'll find the perfect relic to haunt your walls. Dare
+        to enter? Browse our collection… if you dare.
+      </h5>
+    </article>
+    <section class="monster-dropdown"
+             aria-label="Monsters selection dropdown ">
+      <select id="monsterDropdown"
```

Comment:
And all your select elements as well. [Here](https://www.w3schools.com/TAgs/tag_option.asp) you can see an example